app/javascript/menu/side-nav-menu-link.jsx

```diff
@@ -2,6 +2,7 @@ import React, { forwardRef } from 'react';
 import PropTypes from 'prop-types';
 import { SideNavIcon, SideNavItem } from 'carbon-components-react/es/components/UIShell';
 import Link from 'carbon-components-react/es/components/UIShell/Link';
+import { OverflowMenu } from 'carbon-components-react/es/components/OverflowMenu';
```

Comment:
`import OverflowMenu from...`

`node_modules/carbon-components-react/es/components/OverflowMenu/index.js` only exports a default export, not a named one

(should fix the tests :))